src/test/scala/org/camunda/feel/api/StringRepresentationTypeTest.scala

```diff
@@ -121,6 +121,14 @@ class StringRepresentationTypeTest extends AnyFlatSpec with Matchers {
     duration.toString should be("P2M")
   }
 
+  it should "return normalized format " in {
+    ValYearMonthDuration(Period.parse("P2Y")).toString should be("P2Y")
+    ValYearMonthDuration(Period.parse("P24M")).toString should be("P2Y")
+
+    ValYearMonthDuration(Period.parse("P25M")).toString should be("P2Y1M")
+    ValYearMonthDuration(Period.parse("P35M")).toString should be("P2Y11M")
```

Comment:
❓ @saig0, should we also consider cases when both years and months are used in a de-normalized format, such as `P2Y25M`?

Reply:
Sure. I added a test case for it. :+1: